jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -101,51 +113,79 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       container.mixin.readinessProbe.httpGet.withPort(api.config.ports.internal) +
       container.mixin.readinessProbe.httpGet.withScheme('HTTP') +
       container.mixin.readinessProbe.httpGet.withPath('/ready') +
-      if api.config.rbac != {} || api.config.tenants != {} then
-        container.withVolumeMounts(
-          (if api.config.rbac != {} then [
-             {
-               name: 'rbac',
-               mountPath: '/etc/observatorium/rbac.yaml',
-               subPath: 'rbac.yaml',
-               readOnly: true,
-             },
-           ] else []) +
-          (if api.config.tenants != {} then [
-             {
-               name: 'tenants',
-               mountPath: '/etc/observatorium/tenants.yaml',
-               subPath: 'tenants.yaml',
-               readOnly: true,
-             },
-           ] else [])
-        ) else {};
-
-    deployment.new(api.config.name, api.config.replicas, c, api.config.commonLabels) +
-    deployment.mixin.metadata.withNamespace(api.config.namespace) +
-    deployment.mixin.metadata.withLabels(api.config.commonLabels) +
-    deployment.mixin.spec.selector.withMatchLabels(api.config.podLabelSelector) +
-    deployment.mixin.spec.strategy.rollingUpdate.withMaxSurge(0) +
-    deployment.mixin.spec.strategy.rollingUpdate.withMaxUnavailable(1) +
-    if api.config.rbac != {} || api.config.tenants != {} then
-      deployment.mixin.spec.template.spec.withVolumes(
+      container.withVolumeMounts(
         (if api.config.rbac != {} then [
            {
-             configMap: {
-               name: api.config.name,
-             },
              name: 'rbac',
+             mountPath: '/etc/observatorium/rbac.yaml',
```

Comment:
Is this necessary? I don't think we should be exposing also of this in configuration as it makes the API surface extra large and harder to change later IMO